Draft the introduction post for Access.MD (#1)
openmymed/accessmd.github.io: _posts/2020-08-28-acess.md-the-most-open-privacy-first-emr-solution.markdown

---
layout: post

> title: "Access.MD – the most open, privacy first, EMR solution"

Reviewer: Should be Primary Care EMR , since its a huge distinction from specialist EMRs

Also should be ACCESS.MD , all block

Author: ```suggestion
title: "ACCESS.MD – the most open, privacy first,  primary care EMR solution"
```

so something like this ?

date: "2020-08-28 20:18:45 +0300"
author: "Ameer Alqam"
---

## Who we are?

> OpenMyMed is not-for-profit foundation focused on fostering Access.MD, an open source EMR and telemedicine platform, as well as many upcoming OSS projects relating to the medical industry.

Reviewer: We cant say not-for-profit or foundation since we arent formally registered

Author: ```suggestion
OpenMyMed is a foundation focused on fostering Access.MD, an open source EMR and telemedicine platform, as well as many upcoming OSS projects relating to the medical industry.
```

## What is Access.MD?

Access.MD was initially born out of a need to create a patient-doctor intimate direct communication and follow up platform for COVID-19 pandemic, it later then evolved to become a full fledged patient-doctor communication and EMR solution. The primary focus is on openness and privacy, as it’ll be dealing with one of the most personal data a person can have, their medical information!

The primary goal is to manage the lifecycle of hospital/clinic visits, by providing an easy to use way to manage patient data, along with access to it.

This can especially of great value during pandemic times, such as the current COVID-19 pandemic, as it can help in reducing unecessary hospital/clinic visits, in turn reducing the probability of spreading any active pandemic germ, and greatly reduce the risk on frontline workers!

The platform provides mobile and web clients that allow doctors and patiens to follow up on their cases, store their medical data, and provide necessary access to any institute that needs it, given the patient's consent.

## What Access.MD is not

> Acess.MD is not SaaS, although it can be provided by others as SaaS, it is also intended to be realtively easy to self host and maintain. Any clinic, hospital, medical institute, ... etc can host their own Access.MD instance, or a third party can provide it for them for use.

Reviewer: Spelling

Author: ```suggestion
ACCESS.MD is not SaaS, although it can be provided by others as SaaS, it is also intended to be relatively easy to self host and maintain. Any clinic, hospital, medical institute, ... etc can host their own Access.MD instance, or a third party can provide it for them for use.
```
